sdcm/utils/aws_okta.py

```diff
@@ -0,0 +1,80 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2024 ScyllaDB
+
+import os
+import sys
+import pathlib
+import configparser
+import datetime
+import logging
+
+import boto3
+from botocore.exceptions import NoCredentialsError
+from gimme_aws_creds.main import GimmeAWSCreds
+from gimme_aws_creds.ui import CLIUserInterface
+
+account_id = os.environ.get('SCT_AWS_ACCOUNT_ID', '797456418907')
```

Comment:
Is it safe to put the account id here?
Also, which account is it?

Reply:
It's not the best, it's already listed in the code in a few places (and maybe others)

We don't have any other option currently, we need it so we can select the correct one, otherwise user would need to pick it every single time here uses a command

Reply:
https://github.com/search?q=repo%3Ascylladb%2Fscylla-cluster-tests%20797456418907&type=code

Reply:
https://docs.aws.amazon.com/accounts/latest/reference/manage-acct-identifiers.html

> "... While account IDs, like any identifying information, should be used and shared carefully, they are not considered secret, sensitive, or confidential information."